documentcloud.php

```diff
@@ -25,73 +25,104 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
-// Die if this isn't being called from within WordPress.
-defined( 'ABSPATH' ) or die();
-
 class WP_DocumentCloud {
 
+	// Plugin constants
     const CACHING_ENABLED          = true,
           DEFAULT_EMBED_FULL_WIDTH = 940,
           OEMBED_RESOURCE_DOMAIN   = 'www.documentcloud.org',
           OEMBED_PROVIDER          = 'https://www.documentcloud.org/api/oembed.{format}',
           DOCUMENT_PATTERN         = '^(?P<protocol>https?):\/\/www\.documentcloud\.org\/documents\/(?P<document_slug>[0-9]+-[a-z0-9-]+)';
-    
+	/**
+	 * Constructor.
+	 */
     function __construct() {
+		// Check for conflicts with other Documentcloud plugins.
```

Comment:
@bcampeau Mind replacing `Documentcloud` with `DocumentCloud` wherever it appears in your comments?